src/index.html

```diff
@@ -1,13 +1,440 @@
+            class="top-bar__logo"
+          >
+            <img
+              src="./images/logo-dark-2x.png"
+              alt="MyBike logo"
+            >
+          </a>
+
+          <div class="header__icons">
+            <a
+              href="#"
+              class="icon icon--close"
+            >
+            </a>
+          </div>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#"
+                >
+                  Home
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#about-us"
+                >
+                  About us
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#compare-bikes"
+                >
+                  Compare Bikes
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#details"
+                >
+                  Details
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#contacts"
+                >
+                  Contacts
+                </a>
+              </li>
+            </ul>
+          </nav>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__phone-number"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__book-a-test-ride"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </aside>
+    <!-- #endregion -->
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section
+            class="move-free"
+            id="about-us"
+          >
+            <h2 class="
+              move-free__title
+              section-title
+              section-title--align-left
+            ">
+              Move Free
+            </h2>
+
+            <p class="move-free__content">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+            </p>
+          </section>
+
+          <section
+            class="compare"
+            id="compare-bikes"
+          >
+            <h2 class="section-title">Compare Bikes</h2>
+
+            <div class="compare__products">
+              <article class="product compare__product">
+                <a href="#">
+                  <img
+                    src="./images/products/Screenshot 2022-01-14 at 11.12 1.png"
```

Comment:
The name of your media should be understandable. Right now, you have a screenshot, and we don't know what is in this screenshot. Plus, the name of your media shouldn't have empty spaces.